packages/material/scss/pager/_theme.scss

```diff
@@ -0,0 +1,25 @@
+@include exports("pager/theme") {
+    $pager-text: rgba(0, 0, 0, .54) !default;
+
+    .k-pager-wrap {
+        color: $pager-text;
```

Comment:
You can drop this declaration and the `.k-pager-numbers` rule if you include the `_theme` file from theme-default. Bonus: this will also provide the missing hover and focus states.